_search-plugins/search-relevance/compare-query-sets.md

```diff
@@ -2,15 +2,14 @@
 layout: default
 title: Comparing query sets
 nav_order: 12
-parent: Using Search Relevance Workbench
+parent: Search Relevance Workbench
 grand_parent: Search relevance
 has_children: false
-has_toc: false
 ---
 
 # Comparing query sets
 
-This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/17735).    
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/17735).
```

Comment:
You can remove the experimental feature details...  It's now something you have to work to not use if you include it!

Reply:
removed all "experimental tags"